src/main/plan_printer.cpp

```diff
@@ -150,6 +150,10 @@ void OpProfileTree::printOpProfileBoxUpperFrame(uint32_t rowIdx, std::ostringstr
     oss << std::endl;
 }
 
+static std::string dashed_line_account_for_index(uint32_t width, uint32_t indent) {
```

Comment:
use camel case for function names.

Reply:
good news - clang-tidy has a configurable check for identifier casing!